src/EnergyPlus/DXCoils.cc

```diff
@@ -14979,7 +14979,7 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
     bool ErrorsFound(false);
     Real64 FanHeatCorrection;
     Real64 FanPowerCorrection;
-    Real64 FanPowerPerEvapAirFlowRate;
+    Real64 FanPowerPerEvapAirFlowRate = 0.0;
```

Comment:
Debug build showed unit test failed with this variable uninitialized. Maybe my debug switches are different than linux, but initializing here corrected this unit test.